tools/autograd/templates/python_variable_methods.cpp

```diff
@@ -362,7 +362,7 @@ static PyObject * THPVariable_numpy(PyObject* self, PyObject* arg)
         "Can't call numpy() on Variable that requires grad. "
         "Use var.detach().numpy() instead.");
   }
-  return torch::utils::tensor_to_numpy(self_.data());
+  return torch::utils::tensor_to_numpy(self_);
```

Comment:
It's okay to use `self_` instead of `self_.data()` here, because the implementation of `torch::utils::tensor_to_numpy(...)` https://github.com/pytorch/pytorch/blob/9b272affdebf2392d8ede7f83223bf30392a6e43/torch/csrc/utils/tensor_numpy.cpp#L54-L102 doesn't care whether the tensor requires grad or not.